tests/utils/esx/govc.go

```diff
@@ -59,21 +59,32 @@ func GetVMPowerState(vmName string) string {
 // PowerOnVM util powers on the VM
 // govc vm.power -on=true photon
 func PowerOnVM(vmName string) string {
-	// log.Printf("Powering on VM [%s]\n", vmName)
+	log.Printf("Powering on VM [%s]\n", vmName)
 	cmd := esx.PowerOnVM + vmName
 	out, _ := ssh.InvokeCommandLocally(cmd)
 	return out
 }
 
 // PowerOffVM util powers off the VM
+// Caution: If a VM is writing to disk when it receives a
+// Power Off command, data corruption may occur.
 // govc vm.power -off=true photon
 func PowerOffVM(vmName string) string {
-	// log.Printf("Powering off VM [%s]\n", vmName)
+	log.Printf("Powering off VM [%s]\n", vmName)
 	cmd := esx.PowerOffVM + vmName
 	out, _ := ssh.InvokeCommandLocally(cmd)
 	return out
 }
 
+// ShutDownVM util shuts down the VM
+// govc vm.power -s=true photon
+func ShutDownVM(vmName string) string {
+	log.Printf("Shuting down VM [%s]\n", vmName)
+	cmd := esx.ShutDownVM + vmName
```

Comment:
Yeas, but it's still go code calling shell to call local command line (govc) to use govmomi to talk to VC. Why not cut the middleman and call govmomi directly ? It's a generic question to all test, not to your PR - I understand you are just being consistent.

The rest LGTM ; I just want to understand the above